DATA/production/qc-async/pidtof.json

```diff
@@ -0,0 +1,77 @@
+{
+    "qc": {
+        "config": {
+            "database": {
+                "implementation": "CCDB",
+                "host": "ccdb-test.cern.ch:8080",
+                "username": "not_applicable",
+                "password": "not_applicable",
+                "name": "not_applicable"
+            },
+            "Activity": {
+                "number": "42",
+                "type": "2"
+            },
+            "monitoring": {
+                "url": "infologger:///debug?qc"
+            },
+            "consul": {
+                "url": ""
+            },
+            "conditionDB": {
+                "url": "ccdb-test.cern.ch:8080"
+            },
+            "infologger": {
+                "": "Configuration of the Infologger (optional).",
+                "filterDiscardDebug": "false",
+                "": "Set to true to discard debug and trace messages (default: false)",
+                "filterDiscardLevel": "21",
+                "": "Message at this level or above are discarded (default: 21 - Trace)"
+            }
+        },
+        "tasks": {
+            "TaskFT0TOF": {
+                "active": "true",
+                "className": "o2::quality_control_modules::pid::TaskFT0TOF",
+                "moduleName": "QcTOF",
+                "detectorName": "TOF",
+                "cycleDurationSeconds": "10",
+                "maxNumberCycles": "-1",
+                "dataSource": {
+                    "type": "direct",
+                    "query_comment": "checking every matched track",
+                    "query": "matchITSTPCTOF:TOF/MTC_ITSTPC/0;trackITSTPC:GLO/TPCITS/0;trackITSTPCABREFS:GLO/TPCITSAB_REFS/0;trackITSTPCABCLID:GLO/TPCITSAB_CLID/0;tofcluster:TOF/CLUSTERS/0;trackTPC:TPC/TRACKS/0;trackTPCClRefs:TPC/CLUSREFS/0"
+                },
+                "taskParameters": {
+                    "GID": "ITS-TPC,TPC,ITS-TPC-TOF",
+                    "verbose": "false",
+                    "minPtCut": "0.3f",
+                    "etaCut": "0.8f",
+                    "minNTPCClustersCut": "60",
+                    "minDCACut": "100.f",
+                    "minDCACutY": "10.f",
+                    "grpFileName": "o2sim_grp.root",
+                    "geomFileName": "o2sim_geometry.root"
+                },
+                "location": "remote",
```

Comment:
`"location"` is not applicable in this context and will be ignored, you can remove it